doc/src/docs/InputOutputReference/01c-InputOutputReference.md

```diff
@@ -14086,6 +14107,375 @@ AirConditioner:VariableRefrigerantFlow,
+
+#### Field: Outdoor Unit Reference Subcooling 
+
+This numeric field defines the reference subcooling degrees of the outdoor unit. If this field is blank, the default value of 3.0ºC is used. 
+
+#### Field: Refrigerant Temperature Control Algorithm for Indoor Unit
+
+This alpha field specifies the algorithm for the refrigerant temperature control. Two choices are available: ConstantTemp or VariableTemp. The indoor unit evaporating temperature at cooling mode or condensing temperature at heating are fixed in the ConstantTemp algorithm, while in VariableTemp algorithm they can be varied.
+
+#### Field: Reference Evaporating Temperature for Indoor Unit 
+
+This numeric field defines the reference evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is ConstantTemp. If this field is blank, the default value of 6.0ºC is used.
+
+#### Field: Reference Condensing Temperature for Indoor Unit 
+
+This numeric field defines the reference condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is ConstantTemp. If this field is blank, the default value of 44.0ºC is used.
+
+#### Field: Variable Evaporating Temperature Minimum for Indoor Unit
+
+This numeric field defines the minimum evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 4.0ºC is used.
+
+#### Field: Variable Evaporating Temperature Maximum for Indoor Unit
+
+This numeric field defines the maximum evaporating temperature for the indoor unit when VRF runs at cooling mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 13.0ºC is used.
+
+#### Field: Variable Condensing Temperature Minimum for Indoor Unit
+
+This numeric field defines the minimum condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 42.0ºC is used.
+
+#### Field: Variable Condensing Temperature Maximum for Indoor Unit
+
+This numeric field defines the maximum condensing temperature for the indoor unit when VRF runs at heating mode. This field is required if Refrigerant Temperature Control Algorithm is VariableTemp. If this field is blank, the default value of 46.0ºC is used.
+
+#### Field: Outdoor Unit Fan Object Type
+
+This alpha field defines the type of fan in the VRF outdoor unit. The only valid choices are *Fan:OnOff* and *Fan:ConstantVolume*. The input requirements for these fan objects are described elsewhere in this document.
+
```

Comment:
At the beginning, all the fan information at put in the VRF object. These values are used to calculate the fan power.

But then we separate the fan information and put them in a fan object. First, this is more organized for the user inputs. Second, it will be easier to maintain in the future, if the system adds more controls on the OU fan, or the fan is improved to variable speed. Do you think it is fine?